src/index.css

```diff
@@ -1,20 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Serif:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap');
+
 @import 'tailwindcss';
 
-@theme {
-  --color-background: oklch(0.94 0.0335 88.06); /* #F4EAD2 (basic background) */
-  --color-primary: oklch(
-    0.34 0.0713 164.63
-  ); /* #02442F (basic dark text/accent) */
-  --color-secondary: oklch(0.54 0.033 145.1); /* #647564 (secondary text) */
-  --color-light: oklch(0.94 0.0335 88.06); /* #F4EAD2 (light text/elements) */
-
-  --color-success: oklch(
-    0.49 0.1331 149.54
-  ); /* #067332 (success, validation) */
-  --color-error: oklch(0.65 0.2027 35.52); /* #EF5024 (error, danger) */
-  --color-warning: oklch(0.83 0.1693 81.57); /* #FDB90A (warning) */
-
-  --color-primary-light: oklch(
-    0.94 0.0335 88.06
-  ); /* A lighter version of primary */
+@layer base {
+  :root {
+    --color-background: oklch(0.94 0.0335 88.06); /* #F4EAD2 (basic background) */
+    --color-primary: oklch(0.34 0.0713 164.63); /* #02442F (basic dark text/accent) */
+    --color-secondary: oklch(0.54 0.033 145.1); /* #647564 (secondary text) */
+    --color-light: oklch(0.94 0.0335 88.06); /* #F4EAD2 (light text/elements) */
+    --color-success: oklch(0.49 0.1331 149.54); /* #067332 (success, validation) */
+    --color-error: oklch(0.65 0.2027 35.52); /* #EF5024 (error, danger) */
+    --color-warning: oklch(0.83 0.1693 81.57); /* #FDB90A (warning) */
+    --color-primary-light: oklch(0.94 0.0335 88.06); /* A lighter version of primary */
+
+
+    --font-heading: 'IBM Plex Serif', serif;
+    --font-subtext: 'Open Sans', sans-serif;
+  }
+
+  h1 {
```

Comment:
I will have to research this more but I think everything here (after line 22) is replaceable by Tailwind.